resources/sts/4.12/hypershift/ControlPlaneOperatorPolicy.json

```diff
@@ -2,32 +2,89 @@
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Effect": "Allow",
             "Action": [
-                "ec2:CreateVpcEndpoint",
                 "ec2:DescribeVpcEndpoints",
-                "ec2:ModifyVpcEndpoint",
-                "ec2:DeleteVpcEndpoints",
-                "ec2:CreateTags",
-                "route53:ListHostedZones",
-                "ec2:CreateSecurityGroup",
-                "ec2:AuthorizeSecurityGroupIngress",
-                "ec2:AuthorizeSecurityGroupEgress",
-                "ec2:DeleteSecurityGroup",
-                "ec2:RevokeSecurityGroupIngress",
-                "ec2:RevokeSecurityGroupEgress",
-                "ec2:DescribeSecurityGroups",
-                "ec2:DescribeVpcs"
+                "route53:ListHostedZones"
```

Comment:
Can the condition for the list of openshiftapps/devshift urls be added here? 
Also, should this be spun out into two different SIDs? That way the VPC Endpoint describe can have the condition of red-hat-managed: true

Reply:
It doesnt look like the condition key to list specific regex-ed URLs for `ListHostedZones` is supported as per the [docs](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonroute53.html)

I tried adding the red-hat-managed tag condition key for `DescribeVPCEndpoints` but that throws `Client.UnauthorizedOperation` error for the control plane operator when it tries to perform that action. I tried both ResourceTag/RequestTag condition keys. These condition keys don't appear for `DescribeVPCEndpoints` in the [docs](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html) which makes me think this may not work...